src/pdf_to_text.py

```
@@ -1,27 +1,23 @@
 import argparse
+from concurrent.futures import ThreadPoolExecutor
 
-from src.utils.log.log_utils import LogUtils
+from .utils.log.log_utils import LogUtils
 
-from concurrent.futures import ThreadPoolExecutor
 
-from src.utils.log.log_utils import LogUtils
-from src.utils.file.adapters.pdf2image_adapter import PDF2ImageAdapter
-from src.utils.file.adapters.filetype_adapter import FIletypeAdapter
+from .utils.log.log_utils import LogUtils
```

Ruff (0.12.2)

Redefinition of unused `LogUtils` from line 4

Remove definition: `LogUtils`

(F811)